lib/src/component_declaration/component_base.dart

```diff
@@ -368,7 +368,7 @@ abstract class UiComponent<TProps extends UiProps> extends react.Component imple
   /// The parameter may not be `null`.
   @override
   T manageAndReturnTypedDisposable<T extends Disposable>(T disposable) =>
-      _disposableProxy.manageAndReturnTypedDisposable(disposable);
+      _getDisposableProxy().manageAndReturnTypedDisposable(disposable);
```

Comment:
Nice catch! This PR looks great btw, just let me know when it's out of draft and I can do a final review and merge

Reply:
Oh whoops, forgot to take it out of draft mode 😄